grizzly_extras/async_message/mq/rfh2.py

```diff
@@ -122,10 +123,14 @@ def _build_name_values(self) -> None:
             tstamp = str(round(time.time() * 1000))
         else:
             tstamp = self.tstamp
+        metadata_headers = ''
```

Comment:
```
metadata_headers = ''.join([f'<{key}>{value}</{key}>' for key, value in (self.metadata or {}).items()])
```

saves 3 lines of code and 3 levels less of indentation :)

Reply:
nice :) check!